jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/RadioButtonTests.java

```diff
@@ -20,15 +20,109 @@ public void before() {
+        radioButtonSection.seasonRadioButtons.is().displayed();
+        radioButtonSection.winterNGModelRadioButton.is().displayed();
+        radioButtonSection.springNGModelRadioButton.is().displayed();
+        radioButtonSection.summerNGModelRadioButton.is().displayed();
+        radioButtonSection.autumnNGModelRadioButton.is().displayed();
     }
+
+    @Test
+    public void nameBasicRadioButtonTest() {
+        radioButtonSection.firstBasicRadioButton.has().text("Option 1");
+        radioButtonSection.secondBasicRadioButton.has().text("Option 2");
+    }
+
+    @Test
+    public void onlyOneOptionOneButtonEnabledTest() {
+        String enableBasicButton = "mat-radio-button mat-accent cdk-focused cdk-mouse-focused mat-radio-checked";
+        String disableBasicButton = "mat-radio-button mat-accent";
+        radioButtonSection.firstBasicRadioButton.click();
+        radioButtonSection.firstBasicRadioButton.has().attr("class", enableBasicButton);
+        radioButtonSection.secondBasicRadioButton.has().attr("class", disableBasicButton);
+    }
+
+    @Test
+    public void onlyOneOptionTwoButtonEnabledTest() {
+        String enableBasicButton = "mat-radio-button mat-accent cdk-focused cdk-mouse-focused mat-radio-checked";
+        String disableBasicButton = "mat-radio-button mat-accent";
+        radioButtonSection.secondBasicRadioButton.click();
+        radioButtonSection.firstBasicRadioButton.has().attr("class", disableBasicButton);
+        radioButtonSection.secondBasicRadioButton.has().attr("class", enableBasicButton);
+    }
+
+    @Test
+    public void nameSeasonRadioButtonTest() {
+        radioButtonSection.winterNGModelRadioButton.has().text("Winter");
+        radioButtonSection.springNGModelRadioButton.has().text("Spring");
+        radioButtonSection.summerNGModelRadioButton.has().text("Summer");
+        radioButtonSection.autumnNGModelRadioButton.has().text("Autumn");
+    }
+
+    
+    @Test
+    public void favoriteSeasonTest() {
+        radioButtonSection.winterNGModelRadioButton.click();
+        radioButtonSection.favoriteSeason.has().text("Your favorite season is: Winter");
+        radioButtonSection.springNGModelRadioButton.click();
+        radioButtonSection.favoriteSeason.has().text("Your favorite season is: Spring");
+        radioButtonSection.summerNGModelRadioButton.click();
+        radioButtonSection.favoriteSeason.has().text("Your favorite season is: Summer");
+        radioButtonSection.autumnNGModelRadioButton.click();
+        radioButtonSection.favoriteSeason.has().text("Your favorite season is: Autumn");
+    }
+
+
+    @Test
+    public void onlyOneWinterButtonEnabledTest() {
+        String enableSeasonButton = "mat-radio-button example-radio-button mat-accent cdk-focused cdk-mouse-focused mat-radio-checked";
+        String disableSeasonButton = "mat-radio-button example-radio-button mat-accent";
+        radioButtonSection.winterNGModelRadioButton.click();
```

Comment:
Could you please move this two variables to class variables, you use them in four methods.